book/05-distributed-git/sections/contributing.asc

```diff
@@ -1,112 +1,110 @@
-More detailed explanatory text, if necessary.  Wrap it to
-about 72 characters or so.  In some contexts, the first
-line is treated as the subject of an email and the rest of
-the text as the body.  The blank line separating the
-summary from the body is critical (unless you omit the body
-entirely); tools like rebase can get confused if you run
-the two together.
+Bardziej szczegółowy tekst jeżeli jest taka konieczność. Zawijaj
+wiersze po około 72 znakach. Czasami pierwsza linia jest traktowana
+jako temat wiadomości email, a reszta komentarza jako treść. Konieczna
+jest pusta linia oddzielająca podsumowanie od głównej części opisu zmian (chyba
+że całkowicie pominiesz główną część opisu zmian); narzędzia takie jak `rebase`
+mogą się pogubić jeśli ich nie oddzielisz.
 
-Further paragraphs come after blank lines.
+Kolejne paragrafy następują po pustej linii.
 
-  - Bullet points are okay, too
+  - wypunktowania są również poprawne,
 
-  - Typically a hyphen or asterisk is used for the bullet,
-    preceded by a single space, with blank lines in
-    between, but conventions vary here
+  - zazwyczaj myślnik lub gwiazdka jest używana do punktowania,
+    poprzedzona pojedynczym znakiem spacji, z pustą linią pomiędzy,
+    jednak zwyczaje mogą się tutaj różnić.
 -----
 
-If all your commit messages look like this, things will be a lot easier for you and the developers you work with.
-The Git project has well-formatted commit messages – try running `git log --no-merges` there to see what a nicely formatted project-commit history looks like.
+Jeżeli wszystkie Twoje komentarz do zmian będą wyglądały jak ten, współpraca będzie dużo łatwiejsza dla Ciebie i twoich współpracowników.
+Projekt Git ma poprawnie sformatowane komentarze, uruchom polecenie `git log --no-merges` na tym projekcie, aby zobaczyć jak wygląda ładnie sformatowana i prowadzona historia zmian.
 
-In the following examples, and throughout most of this book, for the sake of brevity this book doesn't have nicely-formatted messages like this; instead, we use the `-m` option to `git commit`.
-Do as we say, not as we do.
+W poniższych przykładach, i przez większość tej książki, ze względu na zwięzłość nie sformatowałem treści komentarzy tak ładnie; używam opcji `-m` do `git commit`.
+Rób tak jak mówię, nie tak jak robię.
 
 [[_private_team]]
-==== Private Small Team
+==== Małe prywatne zespoły
 
 (((contributing, private small team)))
-The simplest setup you're likely to encounter is a private project with one or two other developers.
-``Private,'' in this context, means closed-source – not accessible to the outside world.
-You and the other developers all have push access to the repository.
+Najprostszym przykładem który możesz spotkać, to prywatne repozytorium z jednym lub dwoma innymi współpracownikami.
+Jako "prywatne", mam na myśli repozytorium z zamkniętym kodem źródłowym – niedostępnym do odczytu dla innych.
+Ty i inny deweloperzy mają uprawniania do wgrywania ("push") swoich zmian.
+
```

Comment:
is this + needed here?

Reply:
I think you meant + from the next paragraph (diff is misleading) - removed.

Reply:
I think @andre-wojtowicz  that it is because you added an extra line, you can see it in line 103 and 104. There are 2 empty lines, and there should be one empty line.